From f616364f44ea3bc797c826ed9bd71ff306443bc6 Mon Sep 17 00:00:00 2001
From: Alexander Aring <aahringo@redhat.com>
Date: Tue, 21 Mar 2023 09:12:21 -0400
Subject: [PATCH 18/24] dlm_controld: set posix_lock flags to zero

This patch sets another flags variable to zero which is a leftover of
commit 0834ed4b ("dlm_controld: initialize waiter->flags").
---
 dlm_controld/plock.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlm_controld/plock.c b/dlm_controld/plock.c
index ad9b0f2788aa..7f632888793f 100644
--- a/dlm_controld/plock.c
+++ b/dlm_controld/plock.c
@@ -1956,6 +1956,7 @@ void receive_plocks_data(struct lockspace *ls, struct dlm_header *hd, int len)
 			po->pid		= le32_to_cpu(pp->pid);
 			po->nodeid	= le32_to_cpu(pp->nodeid);
 			po->ex		= pp->ex;
+			po->flags	= 0;
 			list_add_tail(&po->list, &r->locks);
 		} else {
 			w = malloc(sizeof(struct lock_waiter));
-- 
2.35.3

